docs/src/main/sphinx/connector/hive.md

```diff
@@ -115,6 +115,16 @@ Hive connector documentation.
 * - `hive.storage-format`
   - The default file format used when creating new tables.
   - `ORC`
+* - `hive.orc.use-column-names`
```

Comment:
For both of these I think we should add a link to the other orc and parquet format config options for completeness.